docs/learn/learn-account-advanced.md

```diff
@@ -9,6 +9,44 @@ slug: ../learn-account-advanced
 
 import RPC from "./../../components/RPC-Connection";
 
+## System Accounts
+
+:::info
+
+You can view system accounts on [Subscan](https://polkadot.subscan.io/account_list?role=module).
+
+:::
+
+As the word suggests, system accounts are used by the system and do not have users. They are used,
+for example, for the treasury, as storage of crowdloan funds, and nomination pools. Those accounts
+do not use the normal transfer process, and they will never issue an
+[extrinsic](./learn-extrinsics.md) since they are not "real" accounts held by users.
+
+The `Account` column in the list of system accounts on
+[Subscan](https://polkadot.subscan.io/account_list?role=module) shows the substrate pallet ID for
+each system account. For example, the treasury pallet id is
+[`py/trsry`](https://github.com/paritytech/polkadot/blob/6282def1bb053858522cf551b86b2d07aad04f29/runtime/polkadot/src/lib.rs#L818).
+
+It is possible to derive account IDs/addresses from these pallet IDs in the same way parachain IDs
+can also be represented as addresses. For example, using the
+[substrate js utilities](https://www.shawntabrizi.com/substrate-js-utilities/) by Shawn Tabrizi,
+converting `py/trsry -> address` will return the substrate address for the treasury:
+
+`5EYCAe5ijiYfyeZ2JJCGq56LmPyNRAKzpG4QkoQkkQNB5e6Z`
+
+This acts as a container for funds but has no actual power over them due to a lack of a
+corresponding key pair. Given the multi-chain property of the ecosystem, such address can be
+mirrored to all parachains so that each parachain has its treasury. At the time of writing, on
```

Comment:
There's no inherent reason that that address would have treasury _logic_ associated with it on a particular chain.